tuf/ngclient/updater.py

```diff
@@ -130,11 +126,16 @@ def refresh(self) -> None:
         specified order (root -> timestamp -> snapshot -> targets) implementing
         all the checks required in the TUF client workflow.
 
-        The metadata for delegated roles are not refreshed by this method as
-        that happens on demand during get_targetinfo().
+        A ``refresh()`` can be done only once during the lifetime of an Updater.
```

Comment:
It seems that I have forgotten, but why can we call refresh only once during the lifetime of an Updater?

Reply:
the issue (#1650) explains it -- with current design allowing a second refresh would be essentially the same as creating a new updater